pliant/manifest.json

```
@@ -0,0 +1,33 @@
+{
+  "display_name": "Pliant",
+  "maintainer": "hello@pliant.io",
+  "manifest_version": "1.0.0",
+  "name": "pliant",
+  "metric_prefix": "pliant.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "IT Process Automation with Pliant.io",
+  "guid": "3beeb950-4020-4e0e-914e-35281dad9719",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-Pliant Integration",
+  "categories": [
+    "orchestration",
+    "notification",
+    "compliance",
+    "automation"
+  ],
+  "type": "check",
+  "is_public": false,
```

```suggestion
  "is_public": true,
```

This will enable pliant integration docs to be generated and available https://docs.datadoghq.com/integrations/